docs/api-docs/api-docs.yml

```diff
@@ -391,7 +391,7 @@ components:
         score_context_improvement:
           type: number
           format: double
-          # TODO: description: "TODO"
+          description: "Difference from the original score"
```

Comment:
@omri374 added description to score_context_improvement - WDYT?

Reply:
Looks good, missed that one, thanks :(